ruby/spec/cucumber/ci_environment/detect_ci_environment_spec.rb

```diff
@@ -3,15 +3,15 @@
 require 'cucumber/ci_environment'
 require 'json'
 
-describe 'detect_ci_environment' do
+describe Cucumber::CiEnvironment, '#detect_ci_environment' do
```

Comment:
I'm not sure how much experience you have with RSpec e.t.c.

Generally the conventions are that you have a top level RSpec.describe block with the name of the class and then inside it you have each method (This way it can be reasonably easy to extend).

Instance methods with `#` and class methods with `.` I think as you know